tests/upgrades-images-unsigned/upgrade_test.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("cOS Upgrade tests - Images unsigned", func() {
 				Expect(out).ToNot(Equal(""))
 
 				version := out
-				out, err = s.Command(fmt.Sprintf("elemental upgrade --no-verify --system.uri docker:%s:cos-system-%s", s.ArtifactsRepo, s.TestVersion))
+				out, err = s.Command(fmt.Sprintf("elemental upgrade --verify --system.uri docker:%s:cos-system-%s", s.ArtifactsRepo, s.TestVersion))
```

Comment:
Isn't this a behavior change? Prior the tests were testing without mtree verification, with this change they will start using verification, so if images are not signed it will fail.

Reply:
argh, you are right, thank you